src/functionalTest/java/uk/gov/hmcts/dts/fact/CourtsEndpointTest.java

```diff
@@ -196,6 +195,7 @@ public void shouldRetrieveCourtDetailBySlugInWelsh() {
 
     @Test
     public void shouldRetrieveAllCourts() {
+        String token = authClient.getToken();
```

Comment:
this should go in an @BeforeAll or something so we don't keep getting new tokens

Reply:
I moved it becuase it's only required for one test